https://github.com/mhinz/neovim-remote/pull/183

From 56d2a4097f4b639a16902390d9bdd8d1350f948c Mon Sep 17 00:00:00 2001
From: oxalica <oxalicc@pm.me>
Date: Sun, 2 Oct 2022 08:09:57 +0800
Subject: [PATCH] Fix tests for neovim 0.8.0

Since neovim/neovim#8519, `NVIM_LISTEN_ADDRESS` is only directly used
when it contains colons or (back-)slashes. Otherwise it is concatanated
with a tempdir prefix and suffixes to form the final path.
--- a/tests/test_nvr.py
+++ b/tests/test_nvr.py
@@ -18,7 +18,7 @@ def run_nvr(cmdlines, env):
         nvr.main(cmdline, env)
 
 def setup_env():
-    env = {'NVIM_LISTEN_ADDRESS': 'pytest_socket_{}'.format(uuid.uuid4())}
+    env = {'NVIM_LISTEN_ADDRESS': './pytest_socket_{}'.format(uuid.uuid4())}
     env.update(os.environ)
     return env
 

